src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,63 @@
+
+Some highlights include:
 
-#### Astro is...
+- **[Islands:](/en/concepts/islands/)** A component-based web architecture optimized for content-driven websites.
+- **[UI-agnostic](/en/core-concepts/framework-components/):** Supports React, Preact, Svelte, Vue, Solid, Lit, HTMX, web components, and more.
+- **[Server-first:](/en/core-concepts/rendering-modes/)** Moves expensive rendering off of your visitors' devices.
+- **[Zero JS, by default:](/en/core-concepts/astro-components/)** Less client-side JavaScript to slow your site down.
+- **[Content collections:](/en/guides/content-collections/)** Organize, validate and provide TypeScript type-safety for your Markdown content.
+- **[Customizable:](/en/guides/integrations-guide/)** Tailwind, MDX, and hundreds of integrations to choose from.
 
-1. [Content-focused](#content-focused): Astro was designed for content-rich websites.
+## Design Principles
+
+Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+
+Astro is...
+
+1. [Content-driven](#content-driven): Astro was designed to showcase your content.
 2. [Server-first](#server-first): Websites run faster when they render HTML on the server.
 3. [Fast by default](#fast-by-default): It should be impossible to build a slow website in Astro.
 4. [Easy to use](#easy-to-use): You don't need to be an expert to build something with Astro.
-5. [Fully-featured, but flexible](#fully-featured-but-flexible): Over 100+ Astro integrations to choose from.
+5. [Developer-focused](#developer-focused): You should have the resources you need to be successful.
 
-## Content-focused
+### Content-driven
 
-**Astro was designed for building content-rich websites.** This includes most marketing sites, publishing sites, documentation sites, blogs, portfolios, and some ecommerce sites.
+**Astro was designed for building content-rich websites.** This includes marketing sites, publishing sites, documentation sites, blogs, portfolios, landing pages, community sites, and ecommerce sites. If you have content to show, it needs to reach your reader quickly.
 
-By contrast, most modern web frameworks are designed for building *web applications*. These frameworks work best for building more complex, application-like experiences in the browser: logged-in admin dashboards, inboxes, social networks, todo lists, and even native-like applications like [Figma](https://figma.com/) and [Ping](https://ping.gg/).
+By contrast, most modern web frameworks were designed for building *web applications*. These frameworks excel at building more complex, application-like experiences in the browser: logged-in admin dashboards, inboxes, social networks, todo lists, and even native-like applications like [Figma](https://figma.com/) and [Ping](https://ping.gg/). However, they often struggle to provide great performance when you need it most.
```

Comment:
The final sentence - `However, they often  struggle to provide ... most` 
Regular Astro Docs Reader POV: Its good 👍 
Twitter Hot Take POV: I kinda feel bad reading it and I would not like it in my feed as well

I have never seen any Framework Docs mentioning how a certain third party product can `struggle during times of need`

> Note: This is a subjective opinion and I don't think my suggestion here would make any difference for better but I want to just talk about it given Fred and Theo(ping.gg creator) have some good content over Youtube & Twitter afaik

Reply:
OK, let's try to avoid hot takes here! I **do** want a however, of some kind. Something that implies: if a thing was built for that stuff, then why would you expect it to excel at the other stuff? Does a change like... "However with that complexity, they CAN struggle to provide great performance when delivering your content." feel any different to you?

Reply:
I will follow up on this in next 8-10 hours (its midnight at my place)
From my PoV the PR is a big 👍

Reply:
I don't know how to mark a comment resolved in GitHub so I will just approve the PR for the better